playground/app/routes/collection.tsx

```diff
@@ -45,9 +45,9 @@ export default function Example() {
 						type: 'radio',
 						options: ['x', 'y', 'z'],
 					}).map((props) => (
-						<label key={props.value} className="inline-block">
+						<label key={props.defaultValue} className="inline-block">
 							<input {...props} />
-							<span className="p-2">{props.value?.toUpperCase()}</span>
+							<span className="p-2">{props.defaultValue.toUpperCase()}</span>
```

Comment:
This is definitely a breaking change now unless we preserve `props.value` 🤔

Reply:
to be honest, I have some ideas on how to improve the collection helper, e.g. sometimes you want the `item` rather than just the `value` so you can provide better labels or more info like a badge or something, but not sure if that's worth doing now.

Reply:
@edmundhung would you want to chat about making these helpers more flexible? I could also create a gist of some ideas if you don't have time to discuss at the moment.

Reply:
I would avoid a breaking change unless it has huge impact at this stage. At least not when v1 is just released 3 months ago 😅 But I would still be happy to learn more about your ideas even there is breaking changes. I am also considering getting rid of all these exports in v2 and provide a CLI that generate these helpers similar to shadcn-ui. Then we don't need to care about breaking changes on these helpers anymore.

Reply:
@edmundhung yeah, maybe we can have a call on discord and do some brain storming if you're available